README.md

```diff
@@ -22,13 +22,71 @@ In addition, a docker-compose file is provided to launch the containers mentione
 * [Wazuh documentation for Docker](https://documentation.wazuh.com/current/docker/index.html)
 * [Docker hub](https://hub.docker.com/u/wazuh)
 
+
 ### Setup SSL certificate and Basic Authentication
 
 Before starting the environment it is required to provide an SSL certificate (or just generate one self-signed) and setup the basic auth.
 
 Documentation on how to provide these two can be found at [nginx_conf/README.md](nginx_conf/README.md).
 
 
+## Environment Variables
+
+Default values are included when available.
+
+### Wazuh
+```
+API_USER="foo"			# Wazuh API username
+API_PASS="bar"			# Wazuh API password
+```
+
+### Elasticsearch
+```
+ELASTIC_CLUSTER="false"					# Setup a cluster
+CLUSTER_NAME="wazuh"					# Cluster name
+CLUSTER_NODE_MASTER="false"				# Set node as master
+CLUSTER_NODE_DATA="true"				# Store data on this node
+CLUSTER_NODE_INGEST="true"				# Setup as ingest node
+CLUSTER_NODE_NAME="wazuh-elasticsearch"                 # Name for this node
+CLUSTER_MASTER_NODE_NAME="master-node"			# Name of the master node
+CLUSTER_MEMORY_LOCK="true"				# Set Elasticsearch memory lock
+CLUSTER_DISCOVERY_SERVICE="wazuh-elasticsearch"         # Set discovery service
+CLUSTER_NUMBER_OF_MASTERS="2"				# Nomber of masters on the cluster
```

Comment:
Line 54 description typo; should be "# Number of masters on the cluster"

Reply:
Fixed in 7a70334. Good catch!